cupy_backends/cuda/cupy_cusparselt.h

```diff
@@ -0,0 +1,32 @@
+#ifndef INCLUDE_GUARD_CUDA_CUPY_CUSPARSELT_H
+#define INCLUDE_GUARD_CUDA_CUPY_CUSPARSELT_H
+
+#include <library_types.h>
+#include <cusparseLt.h>
+
+#if CUSPARSELT_VERSION < 100
+// Added in cuSPARSELt 0.1.0
+
+cusparseStatus_t cusparseLtMatDescriptorDestroy(...) {
+    return CUSPARSE_STATUS_SUCCESS;
```

Comment:
This file is not for RTD/HIP stubs. I feel it may make better sense to return `CUSPARSE_STATUS_NOT_SUPPORTED` instead.